### DISCUSSION TITLE
Job Queue - Print automation

### DISCUSSION THREAD

**Discussion by skydoy** [X1Plus/X1Plus#171] [Ideas]
Is your feature request related to a problem? Please describe.
No

Describe the solution you'd like
I would like to add some kind of list for the jobs pending printing
And a macro for external part release via mqtt 

Describe alternatives you've considered
No Solutions

Software Information
 - Current X1Plus Version:1.0

Additional context
Possibility To automate Printing

**Comment by jwise** [comment]
Hm, I'm not sure I follow.  Can you describe:

* What software would add jobs to a print queue?  Is this something that happens entirely from the printer's front panel, or would slicer changes be involved as well?  If a slicer is involved, is there a standard interface already for this?
* What would external part release be like?  What would this interface look like?  As far as I understand, there are no common hardware solutions for build plate swapping on X1 Carbon right now.

It would probably be most useful to describe, maybe with some mockups, what the entire flow would be like.  For now, I'm going to convert this to a Discussion before it's ready to be an actual feature request, but once this is fleshed out further, this would be cool!

**Reply by skydoy** [reply]
On slicer on my pont of view you dont need any changes.

The idea is to reuse the sd card as paceholder for jobs to be done and maybe add a gcode extra to be able to handle the automation to release parts that would be an external bord with 2 stepper motors 1 for for 1 for part release activated by mqtt waiting for the correct GCode at the end of the print to make the part release

Did i explain well?

**Reply by skydoy** [reply]
i can make  Simple mockup

**Reply by skydoy** [reply]
https://socem-my.sharepoint.com/:u:/g/personal/tiago_febra_socem_pt/EfzBQVkI-v9InGuBuQO3hF8BtGHKiTJnNZGecskuYcnCtw?e=M3N8ur
